lib/iris/tests/integration/test_regridding.py

```diff
@@ -58,23 +53,27 @@ def setUp(self):
     def _regrid(self, method):
         regridder = Regridder(self.src, self.grid, method, 'mask')
         result = regridder(self.src)
-        qplt.pcolor(result, antialiased=False)
-        qplt.plt.gca().coastlines()
+        return result
 
     def test_linear(self):
-        self._regrid('linear')
-        self.check_graphic()
+        result = self._regrid('linear')
+        self.assertEqual(result.shape, (73, 96))
+        self.assertArrayAlmostEqual(result.data.mean(), -16100.351951)
+        self.assertArrayAlmostEqual(result.data.std(), 5603.850769)
 
     def test_nearest(self):
-        self._regrid('nearest')
-        self.check_graphic()
+        result = self._regrid('nearest')
+        self.assertEqual(result.shape, (73, 96))
+        self.assertArrayAlmostEqual(result.data.mean(), -16095.965585)
+        self.assertArrayAlmostEqual(result.data.std(), 5612.657155)
 
 
 @tests.skip_data
-@tests.skip_plot
-class TestGlobalSubsample(tests.GraphicsTest):
+class TestGlobalSubsample(tests.IrisTest):
     def setUp(self):
         self.src = global_pp()
+        _ = self.src.data
```

Comment:
it will, but it can make for messy output during debugging
I tend to prefer to assign to _ as a way to throw this away (so to speak)